src/Games/NexusMods.Games.FOMOD/FomodXmlInstaller.cs

```diff
@@ -40,6 +42,13 @@ public Priority GetPriority(GameInstallation installation, EntityDictionary<Rela
         EntityDictionary<RelativePath, AnalyzedFile> archiveFiles,
         CancellationToken cancellationToken = default)
     {
+
+        // TODO: For now FOMOD installer is enabled for all games and assumes the Game Root folder as the install path.
+        // Consider changing this in the future so games need to opt-in to FOMOD installers and provide a custom install path.
+        var gameTargetPath = gameInstallation.Game.SupportedCapabilities().Values
+            .OfType<AFomodCustomInstallPathCapability>()
+            .FirstOrDefault()?.ModInstallationPath() ?? new GamePath(GameFolderType.Game,"");
```

Comment:
This should do a lookup by ID: `TryGetValue(AFomodCustomInstallPathCapability.CapabilityId, out var capability)`. Afterward, you should do a safe cast: `if (capability is AFomodCustomInstallPathCapability fomodCapability)`. Since this will be used a couple of times, you can make an extension methods out of this:

```csharp
public static bool TryGetCapability<TCapability>(this IDictionary<> capabilities, CapabilityId Id, out TCapability capability);
```